tests/lib/rules/jsx-indent.js

```diff
@@ -51,14 +51,16 @@ ruleTester.run('jsx-indent', rule, {
     ].join('\n'),
     options: [0],
     parserOptions: parserOptions
-  }, {
-    code: [
-      '  <App>',
-      '<Foo />',
-      '  </App>'
-    ].join('\n'),
-    options: [-2],
-    parserOptions: parserOptions
+  // }, {
+  // should we put effort in making this work?
+  // who in their right mind would do this?
```

Comment:
Question stands... It would require more logic in the fixer and I don't think it's worth the complexity.